services/core-api/app/api/mines/response_models.py

```diff
@@ -227,6 +231,16 @@ def format(self, value):
 
 BOND_MODEL = api.model('Bond_guid', {'bond_guid': fields.String})
 
+PERMIT_BASE_MODEL = api.model(
```

Comment:
Could you change the permit model to extend this model then if it is going to be the base model.

Reply:
I think I'll rename it to something else 😄  that is definitely not a base object for `Permit`